Source/utils/string_variant.hpp

```diff
@@ -0,0 +1,77 @@
+#pragma once
+
+#include <string>
+#include "utils/stdcompat/string_view.hpp"
+
+namespace devilution {
+
+class StringVariant
```

Comment:
```suggestion
/**
 * Helper for conversion between various string formats while we are in the process to migrating to purely using string/string_view
 */
class StringVariant
```

Reply:
This wasn't meant as a string conversion tool. (In fact, I'd argue against using it solely for this purpose.) I'll word things differently.